examples/vite-react/src/assets/css/style.scss

```diff
@@ -0,0 +1,7 @@
+@import './partial-style';
```

Comment:
I see the file is `_partial-style.scss` (more `_`). Is this Saas feature or a typo?

Reply:
- It's the feature of Sass,`_` indicates that is a partial sass, so sass will not compile it to a CSS unless we use that in another sass file.
- Also: .sass and .scss solve the same problem, but the syntax is a bit different, .scss has the syntax more similar to .css (contain semicolons, brackets) while .sass doesn't: https://www.geeksforgeeks.org/what-is-the-difference-between-scss-and-sass/#:~:text=SASS%20(Syntactically%20Awesome%20Style%20Sheets,of%20the%20existing%20CSS%20syntax.